sklearn/manifold/t_sne.py

```diff
@@ -142,10 +142,12 @@ def _kl_divergence(params, P, degrees_of_freedom, n_samples, n_components,
 
     # Q is a heavy-tailed distribution: Student's t-distribution
     n = pdist(X_embedded, "sqeuclidean")
-    n += 1.
```

Comment:
I may look at this closer later, but would appreciate someone else's comments on these changes to KL-divergence. @ssaeger, @AlexanderFabisch, @cemoody, @lesteve?